apps/src/lib/node/ledger/shell/prepare_proposal.rs

```diff
@@ -38,15 +45,15 @@ where
 {
     /// Begin a new block.
     ///
-    /// We include half of the new wrapper txs given to us from the mempool
-    /// by tendermint. The rest of the block is filled with decryptions
-    /// of the wrapper txs from the previously committed block.
+    /// Block space is roughly divided into three equal parts, which we call
+    /// allotments. In the following order, each allotted area is home to:
+    /// decrypted, protocol and encrypted transactions, respectively. During
+    /// some protocol phases, we might omit encrypted and decrypted transactions
+    /// entirely, such as when we negotiate DKG parameters.
```

Comment:
nit: would instead link to the documentation for the `block_space_alloc` module, as a single source of truth currently for how this works